ui/components/component-library/tag-url/tag-url.test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable jest/require-top-level-describe */
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { TagUrl } from './tag-url';
+
+describe('TagUrl', () => {
+  it('should render the label inside the TagUrl', () => {
+    const { getByTestId } = render(
+      <TagUrl data-testid="tag-url" label="https://app.uniswap.org" />,
+    );
+    expect(getByTestId('tag-url')).toBeDefined();
+  });
+  it('should render the button if there is a actionButtonLabel inside the tag', () => {
+    render(
+      <TagUrl
+        data-testid="tag-url"
+        label="https://app.uniswap.org"
+        actionButtonLabel="Permissions"
+      />,
+    );
+
+    expect(screen.getByText('Permissions').closest('a')).toBeDefined();
+  });
```

Comment:
Can we improve test coverage here by adding tests for all missing props:
`labelProps`,
`actionButtonProps`,
`src`,
`showLockIcon`,
`avatarFaviconProps`,
`lockIconProps`,
`className`,